main/src/ca/uwaterloo/flix/language/errors/NameError.scala

```diff
@@ -226,8 +219,13 @@ object NameError {
     }
 
     def explain(formatter: Formatter): Option[String] = Some({
-      import formatter._
-      s"${underline("Tip:")} Type variables are always lowercase. Named types are uppercase."
+      """In Flix every type variable begins with a lowercase name. Your type variable
```

Comment:
Another nitpick: everywhere else, `Flix` is kind of personified: "Flix requires XYZ". Here we have a slightly different style where we look at Flix more from the outside.